packages/syft/src/syft/custom_worker/runner_k8s.py

```diff
@@ -230,6 +230,24 @@ def _create_stateful_set(
                                     "image": tag,
                                     "env": env_vars,
                                     "volumeMounts": volume_mounts,
+                                    "livenessProbe": {
+                                        "httpGet": {
+                                            "path": "/api/v2/?probe=livenessProbe",
+                                            "port": 80,
+                                        },
+                                        "initialDelaySeconds": 30,
```

Comment:
don't need "initialDelaySeconds" if we're using startupProbe.

Reply:
Ok. I'll remove it

Reply:
It's still here :D - Meanwhile i'm approving the PR - please merge it once updated